draft-ietf-quic-multipath.md

```diff
@@ -132,18 +132,20 @@ and the actual decision to setup or tear down paths are assumed
 to be handled by the application. But this document does not prevent future extensions from
 defining mechanisms to cope with the remaining scenarios.
 
-Further, this document does not specify detailed algorithms that define
+Further, this document does not specify detailed scheduling algorithms that define
 how multiple, simultaneously open paths are used to send packets.
-As these differ depending on the application requirements,
+As these differ depending on application requirements,
 only some basic implementation guidance is discussed in {{impl-consideration}}.
+However, scheduling algorithms, that may be specified by the IETF in future,
+can be used with the multipath extension specified in this document.
 
```

Comment:
Too many commas? Copy editors insist on no comm before "that", and the one after future is not useful.

Reply:
I thought they help but removed them now.